__DEFINES/global.dm

```diff
@@ -383,6 +383,8 @@ var/list/boss_mobs = list(
 	/mob/living/simple_animal/hostile/wendigo,				// Stupid strong evolving creature things that scream for help
 	/mob/living/simple_animal/hostile/mechahitler,			// Sieg heil!
 	/mob/living/simple_animal/hostile/alien/queen/large,	// The bigger and beefier version of queens.
+	/mob/living/simple_animal/hostile/humanoid/surgeon/boss, //first stage of Doctor Placeholder
```

Comment:
/mob/living/simple_animal/hostile/humanoid/surgeon will take care of both versions